src/java/com/netflix/ice/tag/Tag.java

```diff
@@ -28,8 +28,10 @@ public int compareTo(Tag t) {
     };
 
     public final String name;
+    public final String s3Name;
```

Comment:
Sorry, it has been a long week.  Indeed their is some code that I have forgotten.  Demuxing my large pull request from last year:-)